src/components/IFrame.js

```diff
@@ -30,19 +30,70 @@ function getNewDotURL(url) {
     }
 
     if (pathname === 'policy') {
-        const workspaceID = params.policyID || '';
-        const section = urlObj.hash.slice(1) || 'overview';
-
-        return `workspace/${workspaceID}/${section}`;
+        const workspaceID = params.policyID ?? '';
+        const section = urlObj.hash.slice(1) ?? '';
+
+        let sectionName = section;
+        switch (section) {
+            case '':
+                sectionName = 'overview';
+                break;
+            case 'js_policyEditor_perDiem':
+                sectionName = 'per-diem';
+                break;
+            case 'exportFormats':
+                sectionName = 'export-formats';
+                break;
+            default:
+                sectionName = section;
```

Comment:
2. I suggest removing the empty string case, and setting it in the default case. This could be more intuitive while reading code
```suggestion
        switch (section) {
            case 'js_policyEditor_perDiem':
                sectionName = 'per-diem';
                break;
            case 'exportFormats':
                sectionName = 'export-formats';
                break;
            default:
                sectionName = 'overview';
```